src/storages/pluggable/SegmentsCachePluggable.ts

```diff
@@ -55,38 +88,27 @@ export class SegmentsCachePluggable implements ISegmentsCacheAsync {
   }
 
   /**
-   * Add a list of `segmentKeys` to the given segment `name`.
-   * The returned promise is resolved when the operation success
-   * or rejected with an SplitError if it fails (e.g., wrapper operation fails, JSON parsing error)
-   *
-   * @TODO implement for DataLoader/Producer mode
+   * Add the given segment names to the set of registered segments.
+   * The returned promise is resolved when the operation success,
+   * or rejected with an SplitError if it fails (e.g., wrapper operation fails).
    */
-  addToSegment(name: string, segmentKeys: string[]) {
-    return Promise.resolve(true);
+  registerSegments(segments: string[]) {
```

Comment:
Missing TS return definition

Reply:
Thanks Emi, but I am not defining return types when it is inferred by TS (in this case from the interface the class is implementing), in order to have less verbose code.